### ISSUE TITLE
About runtime WebView2Loader.dll

### ISSUE THREAD

**Issue by chinasmu** [MicrosoftEdge/WebView2Feedback#2822]
**Description**
I want to let my app looks clean so I moved all dlls needed to a filefolder named "System", here is the setting in App.config file:
`<probing privatePath="System"/>`
And then I moved WebView2Loader.dll,Microsoft.Web.WebView2.Core.dll,Microsoft.Web.WebView2.WinForms.dll,Microsoft.Web.WebView2.Wpf.dll
all to System filefolder, and it all works good.
But after version 1.0.1340-pre It can not work anymore(1.0.1305-pre and early version all works good). 
It will throw exception:NullReferenceException.
I found out it is because my app can not found WebView2Loader.dll.
Only after I move WebView2Loader.dll back It can work again.
So I do not know it is a bug or some feature I did not konw before.

[AB#41555487](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/41555487)

**Comment by LiangTheDev** [comment]
I believe that this is due to a behavior change in WebView2 SDK. For your scenario, you could use [CoreWebView2Environment.SetLoaderDllFolderPath](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2environment.setloaderdllfolderpath?view=webview2-dotnet-1.0.1343.22) to specify the path where we can find the WebView2Loader.dll.

**Comment by chinasmu** [comment]
> I believe that this is due to a behavior change in WebView2 SDK. For your scenario, you could use [CoreWebView2Environment.SetLoaderDllFolderPath](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2environment.setloaderdllfolderpath?view=webview2-dotnet-1.0.1343.22) to specify the path where we can find the WebView2Loader.dll.

Yes,Sure.
I use SetLoaderDllFolderPath to specify the path, but there is still a problem.
My scenario:  In my app I will Dynamic create several webView2s according to the need. In general ,there will be 1-8 webView2s.
I Wrapper my webView2 like this:

```
public class MyView : Microsoft.Web.WebView2.WinForms.WebView2
{
     private async void MyView _Init()
            {
                    CoreWebView2Environment.SetLoaderDllFolderPath(Application.StartupPath + @"\System");
                    var env = await CoreWebView2Environment.CreateAsync(null, userDataFolder: $"browserCache\\browser{index}");//index to identify which webView2
                    await EnsureCoreWebView2Async(env);
                }
            }
}
```
So when I start my app, only one browserCache filefolder is generated. That meas only one webView2 is available！
I need to delete " CoreWebView2Environment.SetLoaderDllFolderPath(Application.StartupPath + @"\System");" and put WebView2Loader.dll back to make the app working good.
Anyway I still think there is some problem here.

**Comment by LiangTheDev** [comment]
You should be able to call CoreWebView2Environment.SetLoaderDllFolderPath at the startup of the application. It is a static function that can be called at anytime.
As you stated, it is really a bug. We will try to fix it in the future.

**Comment by champnic** [comment]
This is fixed in the latest prerelease SDK [1.0.1466-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1466-prerelease). Thanks!
